solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -64,16 +57,18 @@ async function fixture() {
   // possible.
   const operators = await registerOperators(
     deployment.randomBeacon as RandomBeacon,
-    (await getUnnamedAccounts()).slice(0, constants.groupSize)
+    deployment.t as T,
+    constants.groupSize,
+    4
```

Comment:
Can we add a comment explaining `4` comes from the number of accounts fetched with `getUnnamedSigners()`? We have this comment in other test files.

Reply:
f931643f8c9330c677ed937b457aed7e45072fe9